src/lib/libast/misc/spawnvex.c

```diff
@@ -696,7 +696,7 @@ pid_t spawnvex(const char *path, char *const argv[], char *const envv[], Spawnve
                              | VEXFLAG(SPAWN_umask)
 #endif
                                  ))
-#if _lib_posix_spawn < 2
+#if _lib_posix_spawn > 0
```

Comment:
Instead of this just simplify it to `#if _lib_posix_spawn`. And in the *features/meson.build* set it to one if the feature test is successful.

Reply:
We actually only want this code path if posix_spawn is broken, so I changed it to `!_lib_posix_spawn` instead.